tests/cases/fourslash/server/autoImportProvider8.ts

```diff
@@ -0,0 +1,68 @@
+/// <reference path="../fourslash.ts" />
+
+// @Filename: /tsconfig.json
+//// { "compilerOptions": { "module": "commonjs" } }
+
+// @Filename: /package.json
+//// { "dependencies": { "mylib": "file:packages/mylib" } }
+
+// @Filename: /packages/mylib/package.json
+//// { "name": "mylib", "version": "1.0.0" }
```

Comment:
The diff between this test and `autoImportProvider7.ts` demonstrates a bug fixed by no longer using `resolveTypeReferenceDirective`:

```diff
- //// { "name": "mylib", "version": "1.0.0", "main": "index.js", "types": "index" }
+ //// { "name": "mylib", "version": "1.0.0" }
```

The test previously needed `"types": "index"` (required to have no extension) to work, due to a quirk of type reference directive resolution.